lighthouse-core/lib/i18n/i18n.js

```diff
@@ -130,10 +130,66 @@ function lookupLocale(locale) {
  * @param {Record<string, string | number>} [values]
  */
 function _preprocessMessageValues(icuMessage, values = {}) {
-  const clonedValues = JSON.parse(JSON.stringify(values));
+  let clonedValues = JSON.parse(JSON.stringify(values));
   const parsed = MessageParser.parse(icuMessage);
+
+  const elements = _collectAllCustomElementsFromICU(parsed.elements);
+
+  clonedValues = _processParsedElements(Array.from(elements.values()), clonedValues);
+
+  return clonedValues;
+}
+
+/**
+ * Function to retrieve all 'argumentElement's from an ICU message. An argumentElement
+ * is an ICU element that has an argument attached like {varName, number, bytes}. This
```

Comment:
```suggestion
 * is an ICU element with an argument in it, like '{varName}' or '{varName, number, bytes}'. This
```
(if that's helpful. I was confused while reading this, thinking this didn't include direct replacement)